cves/kernel/CVE-2019-15666.yml

```diff
@@ -147,10 +147,10 @@ discovered:
 
     If there is no evidence as to how this vulnerability was found, then please
     explain where you looked.
-  answer:
-  automated:
-  contest:
-  developer:
+  answer: 2019-03-01
```

Comment:
Expand on how it is found here. From the patch email https://lore.kernel.org/all/20190228075226.hza3hydllddj65nd@gondor.apana.org.au/T/, the author compiled a kernel with UBSAN, and it caught the bug